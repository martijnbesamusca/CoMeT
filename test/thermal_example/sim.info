{'cmdline': ['../../run-sniper',
             '-v',
             '-s',
             'memTherm_core',
             '-n',
             '4',
             '-c',
             'gainestown_3D',
             '--',
             './test',
             '-p',
             '4'],
 'git_revision': '70a94ea0a46a09195fc7cc97e7411ec90733f6a3',
 'host': '74cff4b09052',
 'pin_version': 'unknown',
 'rusage': (20.485011,
            0.867294,
            220512,
            0,
            0,
            0,
            141263,
            24,
            0,
            19448,
            6960,
            0,
            0,
            0,
            171959,
            376),
 'snipercmd': ['bash',
               '-c',
               '/home/martijn/Documents/uva/thesis/CoMeT/lib/sniper -c /home/martijn/Documents/uva/thesis/CoMeT/config/base.cfg --general/total_cores=4 --general/output_dir=/home/martijn/Documents/uva/thesis/CoMeT/test/thermal_example --config=/home/martijn/Documents/uva/thesis/CoMeT/config/nehalem.cfg --config=/home/martijn/Documents/uva/thesis/CoMeT/config/gainestown_3D.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/martijn/Documents/uva/thesis/CoMeT/test/thermal_example/sim.scripts.py -g --hooks/script0args= -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpe63_he/run_benchmarks'],
 't_elapsed': 21.11323595046997,
 't_start': 1654654412.652829,
 'tracecmds': [['/home/martijn/Documents/uva/thesis/CoMeT/record-trace',
                '-o',
                '/tmp/tmpe63_he/run_benchmarks',
                '-v',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './test',
                '-p',
                '4']],
 'user': None,
 'vmem': 815587328L}
