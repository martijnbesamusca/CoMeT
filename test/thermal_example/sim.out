                                     | Core 0     | Core 1     | Core 2     | Core 3    
  Instructions                       |   20065151 |          0 |          0 |          0
  Cycles                             |    7518960 |    7518960 |    7518960 |    7518960
  IPC                                |       2.67 |       0.00 |       0.00 |       0.00
  Time (ns)                          |    2088600 |    2088600 |    2088600 |    2088600
  Idle time (ns)                     |        -87 |    2088600 |    2088600 |    2088600
  Idle time (%)                      |      -0.0% |     100.0% |     100.0% |     100.0%
Branch predictor stats               |            |            |            |           
  num correct                        |    2466062 |          0 |          0 |          0
  num incorrect                      |      26887 |          0 |          0 |          0
  misprediction rate                 |      1.08% |      0.00% |      0.00% |      0.00%
  mpki                               |       1.34 |       0.00 |       0.00 |       0.00
TLB Summary                          |            |            |            |           
  I-TLB                              |            |            |            |           
    num accesses                     |    3676444 |          0 |          0 |          0
    num misses                       |         43 |          0 |          0 |          0
    miss rate                        |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.00 |       0.00 |       0.00 |       0.00
  D-TLB                              |            |            |            |           
    num accesses                     |    7613568 |          0 |          0 |          0
    num misses                       |        713 |          0 |          0 |          0
    miss rate                        |      0.01% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.04 |       0.00 |       0.00 |       0.00
  L2 TLB                             |            |            |            |           
    num accesses                     |        756 |          0 |          0 |          0
    num misses                       |        377 |          0 |          0 |          0
    miss rate                        |     49.87% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.02 |       0.00 |       0.00 |       0.00
Cache Summary                        |            |            |            |           
  Cache L1-I                         |            |            |            |           
    num cache accesses               |    3676444 |          0 |          0 |          0
    num cache misses                 |       1029 |          0 |          0 |          0
    miss rate                        |      0.03% |       inf% |       inf% |       inf%
    mpki                             |       0.05 |        inf |        inf |        inf
  Cache L1-D                         |            |            |            |           
    num cache accesses               |    7613568 |          0 |          0 |          0
    num cache misses                 |      33147 |          0 |          0 |          0
    miss rate                        |      0.44% |       inf% |       inf% |       inf%
    mpki                             |       1.65 |        inf |        inf |        inf
  Cache L2                           |            |            |            |           
    num cache accesses               |      34663 |          0 |          0 |          0
    num cache misses                 |      32684 |          0 |          0 |          0
    miss rate                        |     94.29% |       inf% |       inf% |       inf%
    mpki                             |       1.63 |        inf |        inf |        inf
DRAM summary                         |            |            |            |           
  num dram accesses                  |      14684 |      14147 |      14225 |      14263
  average dram access latency (ns)   |      31.42 |      23.93 |      23.56 |      24.04
  average dram queueing delay        |       7.99 |       0.50 |       0.13 |       0.61
  average dram bandwidth utilization |      5.00% |      5.00% |      5.00% |      5.00%
Coherency Traffic                    |            |            |            |           
  num loads from dram                |       8200 |          0 |          0 |          0
  num loads from dram cache          |          0 |          0 |          0 |          0
  num loads from remote cache        |          0 |          0 |          0 |          0
